react-app/src/components/Opensearch/main/Filtering/Export/index.tsx

```diff
@@ -54,22 +59,34 @@ export const OsExportData: FC<{
   };
 
   return (
-    <Button
-      variant="outline"
-      onClick={handleExport}
-      disabled={loading}
-      className="w-full xs:w-fit hover:bg-transparent self-center h-10 flex gap-2"
-    >
-      {loading && (
-        <motion.div
-          animate={{ rotate: "360deg" }}
-          transition={{ repeat: Infinity, duration: 0.5 }}
-        >
-          <Loader className="w-4 h-4" />
-        </motion.div>
-      )}
-      {!loading && <Download className="w-4 h-4" />}
-      <span className="prose-sm">Export</span>
-    </Button>
+    <TooltipProvider>
+      <Tooltip disableHoverableContent={true}>
+        <TooltipTrigger asChild className="disabled:pointer-events-auto">
+          <Button
+            variant="outline"
+            onClick={handleExport}
+            disabled={loading || disabled}
+            className="w-full xs:w-fit hover:bg-transparent self-center h-10 flex gap-2"
+            role="tooltip-trigger"
+          >
```

Comment:
If a user were to remove disabled from the attributes and manually enable the button, would `handleExport` still fire off?

Reply:
I believe so since disabled is an optional prop and is only passed through from the search/filtering component so far, but let me know if you think if this doesn't sound right